homeassistant/components/velbus/__init__.py

```diff
@@ -80,6 +80,31 @@ def syn_clock(self, service=None):
 
     hass.services.async_register(DOMAIN, "sync_clock", syn_clock, schema=vol.Schema({}))
 
+    def set_memo_text(service):
+        """Handle Memo Text service call."""
+        memo_text = service.data[CONF_MEMO_TEXT]
+        memo_text.hass = hass
+        try:
+            controller.get_module(service.data[CONF_ADDRESS]).set_memo_text(
```

Comment:
Please extract `service.data[CONF_ADDRESS]` into a local variable.

Reply:
Changed in [149f0c7](https://github.com/home-assistant/core/pull/31222/commits/149f0c7e8216d420fa1a3687ed127e3506a3e267)